### PR TITLE
Add model: motion bert

### PR DESCRIPTION
**Pull request by rebel-jongho**
https://github.com/Walter0807/MotionBERT

MotionBERT is a unified framework designed for learning human motion representations, particularly from video data.

### CODE REVIEW

File: pytorch/vision/pose_estimation/motionbert/compile.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import argparse
+import torch
+import rebel
+
+import urllib.request
+
+sys.path.append(os.path.join(sys.path[0], "MotionBERT"))
+from lib.utils.tools import get_config
```
Comment (rebel-jswoo) [inline pytorch/vision/pose_estimation/motionbert/compile.py:10]: python3 compile.py
Traceback (most recent call last):
  File "/home/jswoo/work/rbln-model-zoo/pytorch/vision/pose_estimation/motionbert/compile.py", line 10, in <module>
    from lib.utils.tools import get_config
ModuleNotFoundError: No module named 'lib'

local test시 위와 같은 에러 발생합니다.

File: pytorch/vision/pose_estimation/motionbert/inference.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import rebel
+
+import numpy as np
+import urllib.request
+
+sys.path.append(os.path.join(sys.path[0], "MotionBERT"))
+from lib.utils.tools import get_config
+from lib.utils.utils_data import flip_data
+from lib.data.dataset_wild import WildDetDataset
+from lib.utils.vismo import render_and_save
```
Comment (rebel-jswoo) [inline pytorch/vision/pose_estimation/motionbert/inference.py:16]: `python3 inference.py
Traceback (most recent call last):
  File "/home/jswoo/work/rbln-model-zoo/pytorch/vision/pose_estimation/motionbert/inference.py", line 16, in <module>
    from lib.utils.vismo import render_and_save
  File "/home/jswoo/work/rbln-model-zoo/pytorch/vision/pose_estimation/motionbert/MotionBERT/lib/utils/vismo.py", line 3, in <module>
    import cv2
  File "/home/jswoo/.local/lib/python3.10/site-packages/cv2/__init__.py", line 181, in <module>
    bootstrap()
  File "/home/jswoo/.local/lib/python3.10/site-packages/cv2/__init__.py", line 153, in bootstrap
    native_module = importlib.import_module("cv2")
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ImportError: libGL.so.1: cannot open shared object file: No such file or directory`

opencv import할때 에러가 나는것같은데, requirments에 추가해야할까요?

Comment (rebel-jswoo) [reply pytorch/vision/pose_estimation/motionbert/inference.py:16]: install해도 안되네요, 구글링 해보니 이런걸 해줘야하는것 같은데,,
$ sudo apt install libgl1-mesa-glx

Comment (rebel-hjkim) [reply pytorch/vision/pose_estimation/motionbert/inference.py:16]: 이 경우 opencv-python-headless 를 설치하면 해결되긴 합니다.
다만 requirements.txt에 이미 있는 ultralytics가 opencv-python 을 dependency로 가지고 있어서, opencv-python 이랑 opencv-python-headless가 동시에 설치되면 아마 문제가 있을 겁니다. (module name이 `cv2`로 서로 동일함)

Comment (rebel-jswoo) [reply pytorch/vision/pose_estimation/motionbert/inference.py:16]: @rebel-junamsong

File: pytorch/vision/pose_estimation/motionbert/inference.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+
+def parsing_argument():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--model_name",
+        type=str,
+        default="base",
+        choices=["base", "lite"],
+        help="(str) type, motionbert 3dpose model_name.",
```
Comment (rebel-shshin) [inline pytorch/vision/pose_estimation/motionbert/inference.py:32]:
```suggestion
        help="(str) type, motionBERT 3D pose model name.",
```

File: pytorch/vision/pose_estimation/motionbert/compile.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+
+def parsing_argument():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--model_name",
+        type=str,
+        default="base",
+        choices=["base", "lite"],
+        help="(str) type, motionbert 3dpose model_name.",
```
Comment (rebel-shshin) [inline pytorch/vision/pose_estimation/motionbert/compile.py:31]:
```suggestion
        help="(str) type, motionBERT 3D pose model_name.",
```

**Review by rebel-junamsong — APPROVED** [review]

**Pull request merged**